include/openthread/backbone_router_ftd.h

```diff
@@ -254,10 +273,12 @@ void otBackboneRouterSetMulticastListenerCallback(otInstance *
 void otBackboneRouterMulticastListenerClear(otInstance *aInstance);
 
 /**
- * This method adds a Multicast Listener.
+ * Adds a Multicast Listener with a timeout value, in seconds. Pass `0` to use
```

Comment:
Split brief and detail into separate blocks.

Reply:
Done.